src/resolvers/platform/WindowsSecurity.js

```diff
@@ -62,7 +62,25 @@ export default {
 
   async applications (root, args, context) {
     const device = await kmd('os', context)
-    const apps = await Device.applications(root, args, context)
+    const DEFAULT_REG_PATH = 'HKLM\\\\Software\\\\Microsoft\\\\Windows\\\\CurrentVersion\\\\Uninstall'
+
+    // gather set of optional registry path overrides from policy
+    const overrides = new Set()
```

Comment:
There's a tension here between specifying `paths.win32` on each application policy object that's having assertions made about it on the one hand, and trying to keep resolver logic general and the data marshaling centralized in Security.js on the other hand. This at least will minimize the number of times we shell out to `reg query` via kmd scripts.

Reply:
Yeah, ideally this could be handled using defaults in the graphQL schema, but I wasn't able to find a way :(
For the mac logic I also mapped and just created the payload needed for the kmd script to fetch the app info (path + name)